src/components/app.js

```diff
@@ -65,6 +66,11 @@ class App extends React.Component {
     }
   };
 
+  getProgram = async programid => {
+    const { ok, sketch } = await fetch.getSketch(programid);
+    return ok && sketch ? true : false;
+  };
+
```

Comment:
Does this pass the sketch information to the routed path? Just noticed that you're not actually returning any sketch information to `MainContainer`.

Reply:
For now it's not passed. Should I just get rid of it or might we need the sketch info later?

Reply:
Right right, I think it'll get passed to the component - that's what I meant by dummy component earlier. Let me see if I can patch together a basic version for this branch.

As a sidenote, means we're probably not going to merge until the feature is done.